tests/x11regressions/gnomecase/nautilus_open_ftp.pm

```diff
@@ -24,6 +24,11 @@ sub run() {
     send_key "ret";
     assert_screen 'nautilus-ftp-suse-com', 20;
 
+    if (get_var('SP2ORLATER')) {
```

Comment:
Hi Oliver, thank you for suggestion:). From main.pm, version_at_least('12-SP2') will set_var('SP2ORLATER', 1), then get_var('SP2ORLATER') should have the same effect. Otherwise, what does it mean to do for set_var('SP2ORLATER'),  just use for other specific usecase?